### ISSUE TITLE
Mobile API Integration: Debts Services (BE Work)

### ISSUE THREAD

**Issue by timwright12** [department-of-veterans-affairs/va-mobile-app#6837]
## User stories (Denise to edit)

**Current Status:** **On Hold**  (Plans for continuance in Q2)
**BE Work Status:** **Completed**
To Continue: FE Journey and Discovery will Needed and Input from Debt Team.  BE Work Set Up Work Completed in Q1. 

MVP:
As a Veteran, I want to see how much money I owe VA. VA calls this "benefit debt and copay bills."
As a Veteran, I want to know how to make a payment OR dispute OR get help.

## mobile api integration (Denise to review)
Mobile API integration to vets-api on:
- Debt Letter Downloader: Downloads debt letters and related information.
- Debts Service: Retrieves debt information for veterans.
- Financial Status Report Downloader: Downloads financial status reports in PDF format.
- Payments Service: Retrieves payment information, including compensation and pension details.

## Debts Services include:

**Level Of Effort "Hard"**

- SharePoint Integration: Uploads and manages PDF files for VA Form 5655. 
- Financial Status Report Service: Manages the submission and creation of financial status reports.

**LOE "Medium"**

- VBS Debt Management Service: Submits VHA FSR (Financial Status Report) via HTTP POST.

LOE "Easy"

- Debt Letter Downloader: Downloads debt letters and related information.
- Debts Service: Retrieves debt information for veterans.
- Financial Status Report Downloader: Downloads financial status reports in PDF format.
- Payments Service: Retrieves payment information, including compensation and pension details.

**Comment by StacyB2023** [comment]
Leaving open per discussion with Rachel. Debt FE work may progress in Q2.

**Comment by StacyB2023** [comment]
@aherzberg FYI moving to Icebox since our BE work is complete but waiting on start of FE work and details from Debt team

**Comment by StacyB2023** [comment]
Closing this epic as our BE ticket work is completed. We may need a new epic if Debt team would like to proceed in future
